intrahost.py

```diff
@@ -794,6 +797,8 @@ def merge_to_vcf(
                     # AF col emitted below, everything excluding the ref allele (one float per non-ref allele)
                     freqs = [(s in iSNVs) and ','.join(map(str, [iSNVs[s].get(a, 0.0) for a in alleles[1:]])) or '.'
                              for s in samplesToUse]
+                    # DP col emitted below
+                    depths = [str(iSNVs_read_depth.get(s, '.')) for s in samples]
```

Comment:
Should be:
```for s in samplesToUse```
(like freqs and nlibs). Some samples get tossed in the filtering process above.